scripts/rebuild-testbench.sh

```diff
@@ -5,27 +5,42 @@
 # fail on any errors
 set -e
 
+# Defaults
+BUILD_TYPE=native
+BUILD_DIR_NAME=build_testbench
+BUILD_TARGET=install
+
 print_usage()
 {
     cat <<EOFUSAGE
-usage: $0 [-f]
+usage: $0 [-f] [-x <platform>]
+       -x Build testbench binary for xt-run for selected platform, e.g. -x tgl
        -f Build testbench with compiler provided by fuzzer
           (default path: $HOME/sof/work/AFL/afl-gcc)
 EOFUSAGE
 }
 
+# die is copy from xtensa-build-all.sh
+die()
+{
+	>&2 printf '%s ERROR: ' "$0"
+	# We want die() to be usable exactly like printf
+	# shellcheck disable=SC2059
+	>&2 printf "$@"
+	exit 1
+}
+
 rebuild_testbench()
 {
     cd "$BUILD_TESTBENCH_DIR"
 
-    rm -rf build_testbench
-
-    mkdir build_testbench
-    cd build_testbench
+    rm -rf $BUILD_DIR_NAME
+    mkdir $BUILD_DIR_NAME
+    cd $BUILD_DIR_NAME
```

Comment:
Please run shellcheck.

Reply:
I've done that - no complains against this code, what issues do you see there?

Reply:
It's not quoted. It looks like `shellcheck` is smart enough to see that it will never be a problem. I would quote it anyway; it could become an absolute path in the future, who knows.

http://mywiki.wooledge.org/Quotes
> When in doubt, double-quote every expansion in your shell commands.